### PR TITLE
feature/544_add_extra_fields_hdfs

### PR DESCRIPTION
**Pull request by frbattid**
* Implements issue #554 
* 100% unit tests passed:
```
Tests run: 59, Failures: 0, Errors: 0, Skipped: 0
```
* (unofficial) e2e tests passed:
```
==========
Cygnus run
==========
--------
json-row
--------
time=2015-10-21T18:33:55.866CEST | lvl=INFO | trans=1445445180-153-0000000099 | function=persistAggregation | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionHDFSSink[805] : [hdfs-sink] Persisting data at OrionHDFSSink. HDFS file (def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room/room1_room.txt), Data ({"recvTimeTs":"1445445231","recvTime":"2015-10-21T16:33:51.319Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room","attrName":"temperature","attrType":"centigrade","attrValue":"0","attrMd":[{"name":"resolution","type":"integer","value":"0"}]}
{"recvTimeTs":"1445445231","recvTime":"2015-10-21T16:33:51.459Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room","attrName":"temperature","attrType":"centigrade","attrValue":"1","attrMd":[{"name":"resolution","type":"integer","value":".50"}]}
{"recvTimeTs":"1445445231","recvTime":"2015-10-21T16:33:51.505Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room","attrName":"temperature","attrType":"centigrade","attrValue":"2","attrMd":[{"name":"resolution","type":"integer","value":"1.00"}]}
...
{"recvTimeTs":"1445445235","recvTime":"2015-10-21T16:33:55.753Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room","attrName":"temperature","attrType":"centigrade","attrValue":"97","attrMd":[{"name":"resolution","type":"integer","value":"48.50"}]}
{"recvTimeTs":"1445445235","recvTime":"2015-10-21T16:33:55.795Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room","attrName":"temperature","attrType":"centigrade","attrValue":"98","attrMd":[{"name":"resolution","type":"integer","value":"49.00"}]}
{"recvTimeTs":"1445445235","recvTime":"2015-10-21T16:33:55.838Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room","attrName":"temperature","attrType":"centigrade","attrValue":"99","attrMd":[{"name":"resolution","type":"integer","value":"49.50"}]}
)
...
...
$ hadoop fs -cat /user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room/room1_room.txt
{"recvTimeTs":"1445445231","recvTime":"2015-10-21T16:33:51.319Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room","attrName":"temperature","attrType":"centigrade","attrValue":"0","attrMd":[{"name":"resolution","type":"integer","value":"0"}]}
{"recvTimeTs":"1445445231","recvTime":"2015-10-21T16:33:51.459Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room","attrName":"temperature","attrType":"centigrade","attrValue":"1","attrMd":[{"name":"resolution","type":"integer","value":".50"}]}
{"recvTimeTs":"1445445231","recvTime":"2015-10-21T16:33:51.505Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room","attrName":"temperature","attrType":"centigrade","attrValue":"2","attrMd":[{"name":"resolution","type":"integer","value":"1.00"}]}
...
{"recvTimeTs":"1445445235","recvTime":"2015-10-21T16:33:55.753Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room","attrName":"temperature","attrType":"centigrade","attrValue":"97","attrMd":[{"name":"resolution","type":"integer","value":"48.50"}]}
{"recvTimeTs":"1445445235","recvTime":"2015-10-21T16:33:55.795Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room","attrName":"temperature","attrType":"centigrade","attrValue":"98","attrMd":[{"name":"resolution","type":"integer","value":"49.00"}]}
{"recvTimeTs":"1445445235","recvTime":"2015-10-21T16:33:55.838Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room","attrName":"temperature","attrType":"centigrade","attrValue":"99","attrMd":[{"name":"resolution","type":"integer","value":"49.50"}]}
-----------
json-column
-----------
time=2015-10-21T18:36:20.776CEST | lvl=INFO | trans=1445445370-985-0000000099 | function=persistAggregation | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionHDFSSink[805] : [hdfs-sink] Persisting data at OrionHDFSSink. HDFS file (def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room/room1_room.txt), Data ({"recvTime":"2015-10-21T16:36:16.474Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room", "temperature":"0", "temperature_md":[{"name":"resolution","type":"integer","value":"0"}]}
{"recvTime":"2015-10-21T16:36:16.573Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room", "temperature":"1", "temperature_md":[{"name":"resolution","type":"integer","value":".50"}]}
{"recvTime":"2015-10-21T16:36:16.615Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room", "temperature":"2", "temperature_md":[{"name":"resolution","type":"integer","value":"1.00"}]}
...
{"recvTime":"2015-10-21T16:36:20.640Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room", "temperature":"97", "temperature_md":[{"name":"resolution","type":"integer","value":"48.50"}]}
{"recvTime":"2015-10-21T16:36:20.685Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room", "temperature":"98", "temperature_md":[{"name":"resolution","type":"integer","value":"49.00"}]}
{"recvTime":"2015-10-21T16:36:20.743Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room", "temperature":"99", "temperature_md":[{"name":"resolution","type":"integer","value":"49.50"}]}
...
...
$ hadoop fs -cat /user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room/room1_room.txt
{"recvTime":"2015-10-21T16:36:16.474Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room", "temperature":"0", "temperature_md":[{"name":"resolution","type":"integer","value":"0"}]}
{"recvTime":"2015-10-21T16:36:16.573Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room", "temperature":"1", "temperature_md":[{"name":"resolution","type":"integer","value":".50"}]}
{"recvTime":"2015-10-21T16:36:16.615Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room", "temperature":"2", "temperature_md":[{"name":"resolution","type":"integer","value":"1.00"}]}
...
{"recvTime":"2015-10-21T16:36:20.640Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room", "temperature":"97", "temperature_md":[{"name":"resolution","type":"integer","value":"48.50"}]}
{"recvTime":"2015-10-21T16:36:20.685Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room", "temperature":"98", "temperature_md":[{"name":"resolution","type":"integer","value":"49.00"}]}
{"recvTime":"2015-10-21T16:36:20.743Z","fiware-servicepath":"def_serv_path_deleteme_1","entityId":"Room1","entityType":"Room", "temperature":"99", "temperature_md":[{"name":"resolution","type":"integer","value":"49.50"}]}
-------
csv-row
-------
time=2015-10-21T18:40:02.679CEST | lvl=INFO | trans=1445445593-699-0000000099 | function=persistAggregation | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionHDFSSink[805] : [hdfs-sink] Persisting data at OrionHDFSSink. HDFS file (def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room/room1_room.txt), Data (1445445598,2015-10-21T16:39:58.441Z,def_serv_path_deleteme_1,Room1,Room,temperature,centigrade,0,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
1445445598,2015-10-21T16:39:58.545Z,def_serv_path_deleteme_1,Room1,Room,temperature,centigrade,1,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
1445445598,2015-10-21T16:39:58.583Z,def_serv_path_deleteme_1,Room1,Room,temperature,centigrade,2,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
...
1445445602,2015-10-21T16:40:02.565Z,def_serv_path_deleteme_1,Room1,Room,temperature,centigrade,97,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
1445445602,2015-10-21T16:40:02.601Z,def_serv_path_deleteme_1,Room1,Room,temperature,centigrade,98,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
1445445602,2015-10-21T16:40:02.641Z,def_serv_path_deleteme_1,Room1,Room,temperature,centigrade,99,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
)
...
...
hadoop fs -cat /user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room/room1_room.txt
1445445598,2015-10-21T16:39:58.441Z,def_serv_path_deleteme_1,Room1,Room,temperature,centigrade,0,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
1445445598,2015-10-21T16:39:58.545Z,def_serv_path_deleteme_1,Room1,Room,temperature,centigrade,1,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
1445445598,2015-10-21T16:39:58.583Z,def_serv_path_deleteme_1,Room1,Room,temperature,centigrade,2,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
...
1445445602,2015-10-21T16:40:02.565Z,def_serv_path_deleteme_1,Room1,Room,temperature,centigrade,97,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
1445445602,2015-10-21T16:40:02.601Z,def_serv_path_deleteme_1,Room1,Room,temperature,centigrade,98,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
1445445602,2015-10-21T16:40:02.641Z,def_serv_path_deleteme_1,Room1,Room,temperature,centigrade,99,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
----------
csv-column
----------
time=2015-10-21T18:42:23.233CEST | lvl=INFO | trans=1445445733-966-0000000099 | function=persistAggregation | comp=Cygnus | msg=com.telefonica.iot.cygnus.sinks.OrionHDFSSink[805] : [hdfs-sink] Persisting data at OrionHDFSSink. HDFS file (def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room/room1_room.txt), Data (2015-10-21T16:42:19.29Z,def_serv_path_deleteme_1,Room1,Room,0,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
2015-10-21T16:42:19.126Z,def_serv_path_deleteme_1,Room1,Room,1,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
2015-10-21T16:42:19.169Z,def_serv_path_deleteme_1,Room1,Room,2,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
...
2015-10-21T16:42:23.105Z,def_serv_path_deleteme_1,Room1,Room,97,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
2015-10-21T16:42:23.146Z,def_serv_path_deleteme_1,Room1,Room,98,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
2015-10-21T16:42:23.189Z,def_serv_path_deleteme_1,Room1,Room,99,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
)
...
...
hadoop fs -cat /user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room/room1_room.txt
2015-10-21T16:42:19.29Z,def_serv_path_deleteme_1,Room1,Room,0,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
2015-10-21T16:42:19.126Z,def_serv_path_deleteme_1,Room1,Room,1,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
2015-10-21T16:42:19.169Z,def_serv_path_deleteme_1,Room1,Room,2,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
...
2015-10-21T16:42:23.105Z,def_serv_path_deleteme_1,Room1,Room,97,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
2015-10-21T16:42:23.146Z,def_serv_path_deleteme_1,Room1,Room,98,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
2015-10-21T16:42:23.189Z,def_serv_path_deleteme_1,Room1,Room,99,hdfs:///user/frb/def_serv_deleteme_1/def_serv_path_deleteme_1/room1_room_temperature_centigrade/room1_room_temperature_centigrade.txt
=================
Migration scripts
=================
--------
json-row
--------
$ hadoop fs -cat /user/soyunpaquete/jsonrow/jsonrow.txt
{"recvTimeTs":"1429535775","recvTime":"2015-04-20T12:13:22.41.124Z","entityId":"car1","entityType":"car","attrName":"speed","attrType":"float","attrValue":"1","attrMd":[]}
{"recvTimeTs":"1429535775","recvTime":"2015-04-20T12:13:22.41.124Z","entityId":"car1","entityType":"car","attrName":"speed","attrType":"float","attrValue":"2","attrMd":[]}
{"recvTimeTs":"1429535775","recvTime":"2015-04-20T12:13:22.41.124Z","entityId":"car1","entityType":"car","attrName":"speed","attrType":"float","attrValue":"3","attrMd":[]}
...
$ hadoop fs -cat /user/soyunpaquete/jsonrow/jsonrow.txt
{"recvTimeTs":"1429535775","recvTime":"2015-04-20T12:13:22.41.124Z","fiware-servicePath":"null","entityId":"car1","entityType":"car","attrName":"speed","attrType":"float","attrValue":"1","attrMd":[]}
{"recvTimeTs":"1429535775","recvTime":"2015-04-20T12:13:22.41.124Z","fiware-servicePath":"null","entityId":"car1","entityType":"car","attrName":"speed","attrType":"float","attrValue":"2","attrMd":[]}
{"recvTimeTs":"1429535775","recvTime":"2015-04-20T12:13:22.41.124Z","fiware-servicePath":"null","entityId":"car1","entityType":"car","attrName":"speed","attrType":"float","attrValue":"3","attrMd":[]}
-----------
json-column
-----------
$ hadoop fs -cat /user/soyunpaquete/jsoncolumn/jsoncolumn.txt
{"recvTime":"2015-04-20T12:13:22.41.124Z","speed":"1","speed_md":[],"oil":"74.6","oil_md":[]}
{"recvTime":"2015-04-20T12:13:22.41.124Z","speed":"2","speed_md":[],"oil":"74.6","oil_md":[]}
{"recvTime":"2015-04-20T12:13:22.41.124Z","speed":"3","speed_md":[],"oil":"74.6","oil_md":[]}
...
$ hadoop fs -cat /user/soyunpaquete/jsoncolumn/jsoncolumn.txt
{"recvTime":"2015-04-20T12:13:22.41.124Z","speed":"1","fiware-servicePath":"null","entityId":"null","entityType":"null","speed_md":[],"oil":"74.6","oil_md":[]}
{"recvTime":"2015-04-20T12:13:22.41.124Z","speed":"2","fiware-servicePath":"null","entityId":"null","entityType":"null","speed_md":[],"oil":"74.6","oil_md":[]}
{"recvTime":"2015-04-20T12:13:22.41.124Z","speed":"3","fiware-servicePath":"null","entityId":"null","entityType":"null","speed_md":[],"oil":"74.6","oil_md":[]}
-------
csv-row
-------
$ hadoop fs -cat /user/soyunpaquete/csvrow/csvrow.txt
1429535775,2015-04-20T12:13:22.41.124Z,car1,car,speed,float,1,hdfs:///user/myuser/vehicles/4wheels/car1_car_speed_float/car1_car_speed_float.txt
1429535775,2015-04-20T12:13:22.41.124Z,car1,car,speed,float,2,hdfs:///user/myuser/vehicles/4wheels/car1_car_speed_float/car1_car_speed_float.txt
1429535775,2015-04-20T12:13:22.41.124Z,car1,car,speed,float,3,hdfs:///user/myuser/vehicles/4wheels/car1_car_speed_float/car1_car_speed_float.txt
...
$ hadoop fs -cat /user/soyunpaquete/csvrow/csvrow.txt
1429535775,2015-04-20T12:13:22.41.124Z,null,car1,car,speed,float,1,hdfs:///user/myuser/vehicles/4wheels/car1_car_speed_float/car1_car_speed_float.txt
1429535775,2015-04-20T12:13:22.41.124Z,null,car1,car,speed,float,2,hdfs:///user/myuser/vehicles/4wheels/car1_car_speed_float/car1_car_speed_float.txt
1429535775,2015-04-20T12:13:22.41.124Z,null,car1,car,speed,float,3,hdfs:///user/myuser/vehicles/4wheels/car1_car_speed_float/car1_car_speed_float.txt
----------
csv-column
----------
$ hadoop fs -cat /user/soyunpaquete/csvcolumn/csvcolumn.txt
2015-04-20T12:13:22.41.124Z,1,hdfs:///user/myuser/vehicles/4wheels/car1_car_speed_float/car1_car_speed_float.txt,74.6,hdfs:///user/myuser/vehicles/4wheels/car1_car_oil_level_float/car1_car_oil_level_float.txt}
2015-04-20T12:13:22.41.124Z,2,hdfs:///user/myuser/vehicles/4wheels/car1_car_speed_float/car1_car_speed_float.txt,74.6,hdfs:///user/myuser/vehicles/4wheels/car1_car_oil_level_float/car1_car_oil_level_float.txt}
2015-04-20T12:13:22.41.124Z,3,hdfs:///user/myuser/vehicles/4wheels/car1_car_speed_float/car1_car_speed_float.txt,74.6,hdfs:///user/myuser/vehicles/4wheels/car1_car_oil_level_float/car1_car_oil_level_float.txt}
...
$ hadoop fs -cat /user/soyunpaquete/csvcolumn/csvcolumn.txt
2015-04-20T12:13:22.41.124Z,1,null,null,null,hdfs:///user/myuser/vehicles/4wheels/car1_car_speed_float/car1_car_speed_float.txt,74.6,hdfs:///user/myuser/vehicles/4wheels/car1_car_oil_level_float/car1_car_oil_level_float.txt}
2015-04-20T12:13:22.41.124Z,2,null,null,null,hdfs:///user/myuser/vehicles/4wheels/car1_car_speed_float/car1_car_speed_float.txt,74.6,hdfs:///user/myuser/vehicles/4wheels/car1_car_oil_level_float/car1_car_oil_level_float.txt}
2015-04-20T12:13:22.41.124Z,3,null,null,null,hdfs:///user/myuser/vehicles/4wheels/car1_car_speed_float/car1_car_speed_float.txt,74.6,hdfs:///user/myuser/vehicles/4wheels/car1_car_oil_level_float/car1_car_oil_level_float.txt}
```
* Assignee @fgalan

### CODE REVIEW

File: doc/design/OrionHDFSSink.md [hunk 1]
```diff
@@ -49,23 +49,33 @@ The context attributes within each context response/entity are iterated, and a o
...
     * `entityId`: Notified entity identifier.
     * `entityType`: Notified entity type.
     * `attrName`: Notified attribute name.
     * `attrType`: Notified attribute type.
     * `attrValue`: In its simplest form, this value is just a string, but since Orion 0.11.0 it can be JSON object or JSON array.
     * `attrMd`: It contains a string serialization of the metadata array for the attribute in JSON (if the attribute hasn't metadata, an empty array `[]` is inserted).
-* `json-column`: A single JSON line is added for all the notified context attributes. This kind of line will contain two fields per each entity's attribute (one for the value, named `<attrName>`, and another for the metadata, named `<attrName>_md`), plus an additional field about the reception time of the data (`recvTime`).
+* `json-column`: A single JSON line is added for all the notified context attributes. This kind of line will contain two fields per each entity's attribute (one for the value, named `<attrName>`, and another for the metadata, named `<attrName>_md`), plus four additional fields:
+    * `recvTime`: The reception time of the data.
+    * `servicePath`: The notified one or default one.
```
Comment (fgalan) [inline doc/design/OrionHDFSSink.md]: Same field is described in a different way under csv-row bullet: "UTC timestamp in human-redable format ([ISO 6801](http://en.wikipedia.org/wiki/ISO_8601)". Same for servicePath.

File: doc/design/OrionHDFSSink.md [hunk 2]
```diff
@@ -49,23 +49,33 @@ The context attributes within each context response/entity are iterated, and a o
...
-* `csv-column`: A single CSV line is added for all the notified context attributes. This kind of line will contain two fields per each entity's attribute (one for the value, named `<attrName>`, and another for the metadata, named `<attrName>_md_file` and containing the name of the HDFS file storing such metadata as explained above), plus an additional field about the reception time of the data (`recvTime`).
+* `csv-column`: A single CSV line is added for all the notified context attributes. This kind of line will contain two fields per each entity's attribute (one for the value, named `<attrName>`, and another for the metadata, named `<attrName>_md_file` and containing the name of the HDFS file storing such metadata as explained above), plus four additional fields:
+    * `recvTime`: The reception time of the data.
```
Comment (fgalan) [inline doc/design/OrionHDFSSink.md]: Same field is described in a different way under csv-row bullet: "UTC timestamp in human-redable format ([ISO 6801](http://en.wikipedia.org/wiki/ISO_8601)". Same for servicePath.

File: resources/cygnus-translators/README.md [hunk 3]
```diff
@@ -1,7 +1,14 @@
-# Cygnus translators
+#<a name="top"></a>Cygnus translators
 
-## `cygnus-translator-0.1-to-0.2.sh`
+Content:
 
+* [`cygnus_translator_0.1_to_0.2.sh`](#section1)
+* [`cygnus_translator_0.1_to_0.3.sh`](#section2)
+* [`cygnus_translator_0.2_to_0.3.sh`](#section3)
+* [`cygnus_translator_0.9.0_to_0.10.0_hdfs.sh`](#section4)
```
Comment (fgalan) [inline resources/cygnus-translators/README.md]: May this name lead to think that if I'm using for instance 0.5.0 (i.e. I'm not using exactly the version name mentioned in the script name) and I want to go to 0.10.0 I do *not* need to follow the procedure?

Maybe the formulation used in Orion could help (https://github.com/telefonicaid/fiware-orion/blob/develop/doc/manuals/admin/install.md#migrating-the-data-stored-in-db): "Upgrading to 0.21.0 and beyond from a pre-0.21.0 version" although not sure how to apply it to a .sh script name.

File: doc/design/OrionHDFSSink.md (hunk 1 above)
Comment (frbattid) [inline doc/design/OrionHDFSSink.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/089d86515cbb8c9ee054ecf910712964b3f73315

File: doc/design/OrionHDFSSink.md (hunk 2 above)
Comment (frbattid) [inline doc/design/OrionHDFSSink.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/089d86515cbb8c9ee054ecf910712964b3f73315

File: resources/cygnus-translators/README.md (hunk 3 above)
Comment (frbattid) [inline resources/cygnus-translators/README.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/089d86515cbb8c9ee054ecf910712964b3f73315

Comment (frbattid) [inline resources/cygnus-translators/README.md]: Re-fixed in https://github.com/telefonicaid/fiware-cygnus/commit/054578b9a04547026fbd86950a4e03a3c5012a42

**Comment by fgalan** [comment]
LGTM

**Pull request merged**